src/rt/sections.d

```diff
@@ -27,6 +27,8 @@ else version (FreeBSD)
     public import rt.sections_elf_shared;
 else version (NetBSD)
     public import rt.sections_elf_shared;
+else version (OpenBSD)
+    public import rt.sections_solaris;
```

Comment:
silly question: why `solaris` and not `elf_shared`?

Reply:
OpenBSD doesn't support everything you're looking for in elf_shared (e.g., no dlinfo, no RTLD_NOLOAD, no dlpi_tls_modid). Faking it caused problems. The Solaris approach worked.

Reply:
In which case can you add a comment here and to the top of ...

Reply:
Sure. Do you mean inline or at the top of this whole block? (I am new to D and its coding style, did my best to follow the conventions I saw.)

Reply:
elf_shared works on Solaris though.  On Solaris 11.0, the tls_modid hidden, but it is attached to the RTLD_DI_LINKMAP (you then have to internally define the private Rt_map and cast the pointer to it).  Solaris 11.4 and later the tls_modid has since been made public.

Reply:
I noticed that with the druntime library you have in GDC. I still am unsure how to operationalize that information though.